FrontEnd/src/components/Header/Navbar/DropdownMenu.module.css

```diff
@@ -22,15 +22,15 @@
 
 .dropdown-menu {
   display: flex;
-  width: 160px;
+  width: 140px;
   padding: 4px 0px;
   flex-direction: column;
   align-items: flex-start;
 
   position: absolute;
   z-index: 1;
-  top: calc(var(--navbar-content-height) + 20px);
-  left: 1140px;
+  top: 150%;
```

Comment:
Does it work properly with different screen sizes? It's better to use static values here

Reply:
Changed to static